src/com/epam/javatrainings/classdecomposition/pizzeria_palmetto/services/IngredientsCreator.java

```diff
@@ -0,0 +1,5 @@
+package com.epam.javatrainings.classdecomposition.pizzeria_palmetto.services;
+
+public interface IngredientsCreator {
+    boolean addIngredient(String ingredient);
```

Comment:
"String ingredient" should be replaced with "Ingredients ingredient"

Reply:
done